documentation/sphinx/source/release-notes.rst

```diff
@@ -47,10 +49,12 @@ Fixes
 * Databases with more than 10TB of data would pause for a few seconds after recovery. [6.0.6] `(PR #705) <https://github.com/apple/foundationdb/pull/705>`_
 * Sometimes a minority of coordinators would fail to converge after a new leader was elected. [6.0.6] `(PR #700) <https://github.com/apple/foundationdb/pull/700>`_
 * Calling status too many times in a 5 second interval caused the cluster controller to pause for a few seconds. [6.0.7] `(PR #711) <https://github.com/apple/foundationdb/pull/711>`_
-* Configuring from usable_regions=2 to usable_regions=1 on a cluster with a large number of processes would prevent data distribution from completing. [6.0.8] `(PR #721) <https://github.com/apple/foundationdb/pull/721>`_
+* Configuring from usable_regions=2 to usable_regions=1 on a cluster with a large number of processes would prevent data distribution from completing. [6.0.8] `(PR #721) <https://github.com/apple/foundationdb/pull/721>`_ [6.0.10] `(PR #739) <https://github.com/apple/foundationdb/pull/739>`_
```

Comment:
If this wasn't fully fixed in 6.0.8, then we may just want to list 6.0.10 as the release that fixes it. If it was better (from a user's perspective) in 6.0.8, though, and then improved further in 6.0.10, then I could see listing both.

Reply:
I think because of the lack of yields, it would have been still broken on 6.0.8, because it would require the knob change to stop the failure monitor, so I will just go with 6.0.10